siliconcompiler/core.py

```diff
@@ -1163,17 +1163,17 @@ def _find_files(self,
             dependencies.append(None)
 
         for (dependency, path) in zip(dependencies, paths):
+            if not search_paths:
```

Comment:
Why would you give this higher priority?

Reply:
because the imported_file_paths need higher priority